Cookbook/Proposals/Template_Proposal_Meeting_Invitation.md

```diff
@@ -1,27 +1,32 @@
 Hi all,
 
-This meeting is to discuss the "<Proposal Title>" proposal authored by <Proposal's Author>.
+This meeting is to discuss the "<#Proposal Title#>" proposal authored by <#Proposal's Author#>.
```

Comment:
Had to add those because otherwise GitHub sees `<xx yy>` as if it were an HTML tag and doesn't render the text inside, so that part was not rendered and thus not copied during copy/paste in Outlook